Wikipedia/Code/SectionEditorViewController.m

```diff
@@ -323,6 +323,7 @@ - (void)scrollTextViewSoCursorNotUnderKeyboard:(UITextView *)textView {
 - (void)didReceiveMemoryWarning {
     [super didReceiveMemoryWarning];
     // [self.webView toggleRichEditor];
+    // [self.webView becomeFirstResponder];
```

Comment:
@nambatee if you uncomment this, then on my buttons subclass if you select some text, then tap the text area to steal focus, then trigger mem warning, you should see the web view text be re-selected :)